mongo/with_transactions_test.go

```diff
@@ -259,6 +259,118 @@ func TestConvenientTransactions(t *testing.T) {
 		assert.True(t, ok, "expected context for abortTransaction to contain ctxKey")
 		assert.Equal(t, "foobar", ctxValue, "expected value for ctxKey to be 'world', got %s", ctxValue)
 	})
+	t.Run("commitTransaction timeout allows abortTransaction", func(t *testing.T) {
+		// Create a special CommandMonitor that only records information about abortTransaction events and also
+		// records the Context used in the CommandStartedEvent listener.
+		var abortStarted []*event.CommandStartedEvent
+		var abortSucceeded []*event.CommandSucceededEvent
+		var abortFailed []*event.CommandFailedEvent
+		var abortCtx context.Context
```

Comment:
This was necessary for the previous test, but is not needed here. It should be removed along with the reference to it in the comment at the top of the test.

Reply:
Ah sounds good! Removed both.